config/cluster-install/controller-manager-deploy.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          env:
+            - name: ARGO_DATAFLOW_CLUSTER
+              value: quick-start
```

Comment:
this is wrong, user must specify

Reply:
So User needs to edit the YAML before applying it.

Reply:
Can we have it default for `quick-cluster-start.yaml` file?